MatrixAlgorithms/Matrix2d.cpp

```diff
@@ -135,6 +139,81 @@ void Matrix2d::operator*=(double number)
     }
 }
 
+
+//operator overloading for matrix multiplication (*)
+Matrix2d  Matrix2d::operator * (const Matrix2d& mat2) 
```

Comment:
Check and correct the indentation in all files. In this project tab size = 4 spaces.